### PR TITLE
circuit Stonks & Plenbit demo

### PR DESCRIPTION
**Pull request by ladyada**
2 demoz!

### CODE REVIEW

File: CLUE_plenbit_demo/code.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+i2c = busio.I2C(board.SCL, board.SDA)
+pca = PCA9685(i2c, address=0x68)
+pca.frequency = 50
+led = DigitalInOut(board.P16)
+led.direction = Direction.OUTPUT
+led.value = True
+
+while True:
+    pass
```
Comment (jepler) [inline CLUE_plenbit_demo/code.py]: won't this stop it from actually doing anything?  Is that intentional?

**Comment by ladyada** [comment]
w00ps fixed

**Comment by ladyada** [comment]
@jepler can u rereview?

File: Temperature_GIF_Player/Temperature_GIF_Player.ino [hunk 2]
```diff
@@ -0,0 +1,335 @@
+// please read credits at the bottom of file
```
Comment (jepler) [inline Temperature_GIF_Player/Temperature_GIF_Player.ino]: this file was added to this PR by accident?

Comment (ladyada) [reply Temperature_GIF_Player/Temperature_GIF_Player.ino]: nah i have 3 guides im workin on

**Comment by ladyada** [comment]
thanx!

**Pull request merged**